src/Lib/Mail/Mail.php

```diff
@@ -0,0 +1,105 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Lib\Mail;
+
+class Mail
+{
+
+    /** @var string */
+    private $headders;
+
+
+    /**
+     *
+     * @param string $to
+     * @param string $from
+     * @param string $fromName
+     * @param string $content
+     */
+    public function __construct(string $to, string $from, string $fromName, string $subject, string $content)
+    {
+        $this->to = $to?$to:false;
+        $this->from = testEmail($from, 'From')?$from:false;
+        $this->fromName = $fromName?$fromName:false;
+        $this->subject = $subject?$subject:false;
+        $this->content = $content?$content:false;
+    }
+
+
+    /**
+    * Add hidden copy
+    *
+    * @return void
+    */
+    public static function addBcc($bcc):void
+    {
+        $this->bcc = testEmail($bcc, 'Bcc')?$bcc:false;
+    }
+
+
+    /**
+    * Add reply to (different address)
+    *
+    * @return void
+    */
+    public static function addReplyTo($replyTo, $replyToName):void
```

Comment:
I am suggesting to not implement this feature. Can lead to problem with e-mail validity.

Reply:
What do you mean? You think it's not user friendly to reply to different @?

Reply:
It is user friendly, but setting reply to to different address then sender can lead to bad "spam score". I think if user have e-mail account like info@domain.com the reply to should be same. Otherwise we can setup e-mail redirect.

Reply:
true, will cancel that